ci-operator/step-registry/cucushift/hypershift-extended/enable-dns/update-record/cucushift-hypershift-extended-enable-dns-update-record-commands.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#Get controlplane endpoint
+if [ ! -f "${SHARED_DIR}/kubeconfig" ]; then
+    exit 1
+fi
+export KUBECONFIG="${SHARED_DIR}/kubeconfig"
+if [[ -f "${SHARED_DIR}/proxy-conf.sh" ]]; then
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+CLUSTER_NAME=$(oc get hostedclusters -n "$HYPERSHIFT_NAMESPACE" -o jsonpath='{.items[0].metadata.name}')
+echo "hostedclusters => ns: $HYPERSHIFT_NAMESPACE , cluster_name: $CLUSTER_NAME"
+if ! KAS_DNS_NAME=$(oc get "hostedclusters/${CLUSTER_NAME}" -n "${HYPERSHIFT_NAMESPACE}" \
+    -o jsonpath='{.spec.kubeAPIServerDNSName}' 2>/dev/null)
+then
+    echo "ERROR: HostedCluster '${CLUSTER_NAME}' not found in namespace '${HYPERSHIFT_NAMESPACE}'" >&2
+    exit 1
+elif [[ -z "${KAS_DNS_NAME}" ]]; then
+    echo "ERROR: KubeAPI Server DNS name not configured for '${CLUSTER_NAME}'" >&2
+    exit 1
+fi
+CP_EP=$(oc get hostedclusters/${CLUSTER_NAME} -n "$HYPERSHIFT_NAMESPACE" -o jsonpath='{.status.controlPlaneEndpoint.host}')
+
+#Update route53 record value
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
```

Comment:
if it is only available on the AWS platform, could you add the file name with some words like "update-aws-record"?